config/application.rb

```diff
@@ -15,5 +15,7 @@ class Application < Rails::Application
     # Application configuration can go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded after loading
     # the framework and any gems in your application.
+    config.action_view.form_with_generates_remote_forms=false
```

Comment:
what does this line do?

Reply:
Logan can speak more to it, I believe it removes some need to include `local: true` on various buttons or links...